src/components/playback/mediasession.js

```diff
@@ -49,121 +30,96 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
         return null;
     }
 
-    function imageUrl(item, options) {
-
-        options = options || {};
-        options.type = options.type || "Primary";
```

Comment:
same, what if `options.type` is unset? before it defaulted it to `Primary` but now it would remain unset